### ISSUE TITLE
Inconsistent values in 2020 Hood County general source file

### ISSUE THREAD

**Issue by warwickmm** [openelections/openelections-data-tx#373]
The [source file](https://github.com/openelections/openelections-sources-tx/blob/0894ee5e2ed07bd98dbe0c4181166ae1caa17583/2020/general/HOOD_COUNTY-2020_NOVEMBER_3RD_GENERAL_ELECTION_1132020-November%203%202020%20pct%20by%20pct.pdf) for the 2020 Hood County general results contains values that are inconsistent.  It is often the case where the sum of the components (mail, early voting, election day, provisional) is not equal to the total votes.  

For example, the total ballots cast should be `288 + 1809 + 474 + 16 = 2587` instead of `2585`.  In fact, many of the totals appear to be off by `2`.

```
Headers: ['county', 'precinct', 'office', 'district', 'candidate', 'party', 'votes', 'mail', 'early_voting', 'election_day', 'provisional']:
Row 479: ['Hood', 'Precinct 2', 'Ballots Cast', '', '', '', '2585', '288', '1809', '474', '16']
Row 480: ['Hood', 'Precinct 2', 'President / Vice President', '', 'Donald J. Trump / Michael R. Pence', 'REP', '2002', '165', '1434', '396', '9']
Row 487: ['Hood', 'Precinct 2', 'U.S. Senate', '', 'John Cornyn', 'REP', '1949', '170', '1396', '375', '10']
Row 494: ['Hood', 'Precinct 2', 'U.S. House', '11', 'August Pfluger', 'REP', '1941', '154', '1393', '386', '10']
Row 496: ['Hood', 'Precinct 2', 'U.S. House', '11', 'Wacey Alpha Cody', 'LIB', '53', '2', '31', '20', '2']
Row 499: ['Hood', 'Precinct 2', 'Railroad Commissioner', '', 'James "Jim" Wright', 'REP', '1937', '166', '1387', '376', '10']
Row 505: ['Hood', 'Precinct 2', 'State Senate', '22', 'Brian Birdwell', 'REP', '2012', '163', '1440', '399', '12']
Row 509: ['Hood', 'Precinct 2', 'State Representative', '60', 'Glenn Rogers', 'REP', '2164', '187', '1539', '428', '12']
```

![image](https://user-images.githubusercontent.com/17345532/133302763-3589e99c-5ba2-41e4-972f-07d8689b34a2.png)

**Comment by warwickmm** [comment]
Is there another source for this, or is this something we would want to reach out to the county for clarification?

**Comment by dwillis** [comment]
@warwickmm Probably need to talk to the county for clarification. Thanks for surfacing this.

**Comment by warwickmm** [comment]
No problem.  The vote breakdown totals test is turning out to be more useful than I anticipated.

**Comment by dwillis** [comment]
I am both happy and sad about this.

**Comment by warwickmm** [comment]
Yes.  I'm also afraid that since it's difficult to test for totals that are greater than the component sum, we're only catching half of these types of issues.

**Comment by JCinTX** [comment]
Note that the totals for President in this example shows Overvotes = 2
(apparently both in mail ballots.) Need to clarify with the county how that
may occur and how they handle it in computing the total votes.

Rather than recounting all the ballots it's possible they ignore it if the
overvotes do not affect the election outcome.

This is not an unusual occurrence. I've seen it in every election in my
county (Collin).

Undervotes, on the other hand, are easier to explain. Not everyone casts a
vote for every office on the ballot.
